core/src/avm2/globals/XMLList.as

```diff
@@ -22,6 +22,7 @@ package {
         AS3 native function toXMLString():String;
         AS3 native function toString():String;
         AS3 native function comments():XMLList;
+        AS3 native function parent():Object;
```

Comment:
I'm pretty sure this is also defined as a method on the `prototype`

Reply:
Also, the return type should be `*`, to allow for returning `undefined` (also, that's the signature in playerglobals)

Reply:
Fixed.